packages/sui-react-initial-props/src/ssrComponentWithInitialProps.js

```diff
@@ -9,18 +9,34 @@ export default function ssrComponentWithInitialProps ({
   context,
   renderProps
 }) {
+  const startGetInitialProps = process.hrtime()
   // get the page with the initialProps
   const pageComponent =
     renderProps.components[renderProps.components.length - 1]
   // use the getInitialProps from the page to retrieve the props to initialize
   const getInitialProps = pageComponent.getInitialProps || EMPTY_FUNC
   return getInitialProps(context).then(initialProps => {
+    const diffGetInitialProps = process.hrtime(startGetInitialProps)
+    const timeGetInitialProps =
+      diffGetInitialProps[0] * 1E3 + diffGetInitialProps[1] * 1e-6
```

Comment:
Extract this to a method as it's exactly the same operation as 32 and it doesn't seem trivial.

Reply:
Ok